buildspecs/linux_ppc-64.spec

```diff
@@ -96,7 +96,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<property name="uma_make_cmd_cxx_exe_ld" value="$(CC)"/>
 		<property name="uma_make_cmd_dll_ld" value="$(CC)"/>
 		<property name="uma_make_cmd_exe_ld" value="$(CC)"/>
-		<property name="uma_make_cmd_ppc_gcc_cxx" value="gcc-4.6"/>
```

Comment:
This should be deleted.

Reply:
If  uma_make_cmd_ppc_gcc_cxx is deleted then PPC_GCC_CXX would be set to xlc (mkconstants.mk.ftl). Shouldn't BytecodeInterpreter be compiled with gcc?

Reply:
yup